express-api/src/services/ltsa/ltsaServices.ts

```diff
@@ -0,0 +1,120 @@
+import { ErrorWithCode } from '@/utilities/customErrors/ErrorWithCode';
+import {
+  ILtsaTitleSummaryModel,
+  ILtsaTitleSummaryResponse,
+} from './interfaces/ILtsaTitleSummaryModel';
+import { ILtsaTokens } from '@/services/ltsa/interfaces/ILtsaTokens';
+import { ILtsaOrder } from '@/services/ltsa/interfaces/ILtsaOrder';
+import getConfig from '@/constants/config';
+
+const ltsaConfig = getConfig().ltsa;
+export const processLTSARequest = async (pid: string) => {
+  // make a request to get an access token from LTSA
+  const ltsatoken = await getTokenAsync();
+
+  // make a request to get title summary info from LTSA
+  const titleSummaryResponse: ILtsaTitleSummaryResponse = await getTitleSummary(
+    ltsatoken.accessToken,
+    pid,
+  );
+  // Access the first title summary in the list
+  const titleSummary: ILtsaTitleSummaryModel = titleSummaryResponse.titleSummaries[0];
```

Comment:
Why does getTitleSummary return an array? Is it sufficient to just take the first element whatever that may be?

Reply:
> Why does getTitleSummary return an array? Is it sufficient to just take the first element whatever that may be?

There are often cases where multiple owners are registered on a title . eg) joint tenancy or people who have fractional interest in properties.So the title summary brings in more than one ownership detail. (refer attached image)
![multipleowners](https://github.com/bcgov/PIMS/assets/80914899/2554059a-c28b-409c-822f-2aeaec54b2d3)

I'm taking the first element to extract title number and land district code to be displayed in the parcel info page. Since, title number and land district code are same across all ownership, I'm just getting the values from the first element.